### PR TITLE
fix(minor): get only transporter in link field

### PR DESCRIPTION
**Pull request by rehanansari26**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

**Comment by rehanansari26** [comment]
@s-aga-r

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.js [hunk 1]
```diff
@@ -57,6 +57,12 @@ frappe.ui.form.on("Purchase Receipt", {
 				}
 			}
 		});
+		
+		frm.set_query("transporter", function() {
```
Comment (s-aga-r) [inline erpnext/stock/doctype/purchase_receipt/purchase_receipt.js:61]: @rehanansari26 There is no field named `transporter` in PR. There is `transporter_name` which is a Data field, so filters can't applied to it using set_query.

**Review by s-aga-r — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/purchase_receipt/purchase_receipt.js (hunk 1 above)
Comment (rehanansari26) [reply erpnext/stock/doctype/purchase_receipt/purchase_receipt.js:61]: @s-aga-r  then it is created after installation? what is mean by **Is System Generated** ? It is belong to which app ?
image

Comment (s-aga-r) [reply erpnext/stock/doctype/purchase_receipt/purchase_receipt.js:61]: It might be created by the India Compliance App, need to check.

**Pull request closed without merging**